src/alpinelinux.ipxe

```diff
@@ -0,0 +1,29 @@
+#!ipxe
+
+# Alpine Linux netboot.xyz ipxe script.
+# https://alpinelinux.org
+
+goto ${menu}
+
+:alpinelinux
+set os Alpine Linux
+iseq ${arch} x86_64 && set bootarch x86_64 || set bootarch x86
+
+menu ${os} [${bootarch}]
+item latest-stable Latest stable
+item edge Edge (development)
+choose version || goto alpine_exit
+goto boot
+
+:boot
+set base-url http://dev.alpinelinux.org/~clandmeter/netboot/${version}/${bootarch}
```

Comment:
I'm hesitant to add unofficial images but I'm ok with treating the distro as experimental as long as they are hosted from the alpinelinux.org domain.  If you can make progress on official images, we can remove the experimental tag and add signature checking for the images.  That way we can at least let the users experiment with the distro.